microsoft/PowerToys
# [FancyZones] Set correct corner preference when snapping windows on Windows 11

## Summary of the Pull Request

**What is this about:**
When snapping a window on Windows 11 the app retains the rounded corners. This does not happen when using native Windows 11 snap.

**What is included in the PR:** 
Working example where apps get square borders when snapped and rounded when un-snapped.

**How does someone test / validate:** 
* Run Powertoys on Windows 11
* Open Notepad (notice rounded corners)
* Snap Notepad to a zone
* Notepad should now have square corners
* Unsnap Notepad
* Notepad should now have rounded corners again

## Quality Checklist

- [x] **Linked issue:** #11991, #12787
